static/app/views/ai-analytics/landing.tsx

```diff
@@ -0,0 +1,25 @@
+import * as Layout from 'sentry/components/layouts/thirds';
+import {PageHeadingQuestionTooltip} from 'sentry/components/pageHeadingQuestionTooltip';
+import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
+import {t} from 'sentry/locale';
+import useOrganization from 'sentry/utils/useOrganization';
+
+export default function AiAnalyticsPage() {
+  const organization = useOrganization();
+
+  return (
+    <SentryDocumentTitle title={`AI Analytics — ${organization.slug}`}>
+      <Layout.Header>
+        <Layout.HeaderContent>
+          <Layout.Title>
+            {t('AI Analytics')}
+            <PageHeadingQuestionTooltip
+              title={t('View analytics and information about your AI pipelines')}
+              docsUrl="https://docs.sentry.io/product/ai-analytics/"
```

Comment:
I assume this docs link is a TODO?

Reply:
for now, yes - will be created for the alpha